packages/liveblocks-node/src/client.ts

```diff
@@ -48,6 +53,46 @@ type ThreadParticipants = {
   participantIds: string[];
 };
 
+type Permission = "room:write" | "room:read" | "room:presence:write";
+type RoomAccesses = Record<string, Permission[]>;
+type RoomMetadata = Record<string, string | string[]>;
+
+type RoomInfo = {
```

Comment:
We need to re-export these types so our users can use them in their own functions.